### PR TITLE
"kind" property values

### PR DESCRIPTION
**Pull request by seligj95**
updated based on testing, removed api app since that is no longer relevant

### CODE REVIEW

**Comment by seligj95** [comment]
@btardif - it won't let me add you as a reviewer so tagging you here instead

**Comment by btardif** [comment]
@ehamai can you take a look here see if there is anything we are missing?

File: Things_You_Should_Know/kind_property.md [hunk 1]
```diff
@@ -16,13 +16,15 @@ You will need to manually set the **kind property** if you are creating resource
...
-|`hyperV`            | Windows Container Web App      |
+|Kind                      | Resource Type                  |
+|--------------------------|--------------------------------|
+|`app`                     | Windows Web app                |
+|`app,linux`               | Linux Web app                  |
+|`hyperV`                  | Windows Container Web App      |
+|`app,container,windows`   | Windows Container Web App      |
+|`app,linux,container`     | Linux Container Web App        |
+|`functionapp`             | Function App                   |
+|`functionapp,linux`       | Linux Consumption Function app |
```
Comment (ehamai) [inline Things_You_Should_Know/kind_property.md:27]: You missed a couple less known ones I think.

functionapp,linux,container,kubernetes - Function container app on ARC
functionapp,linux,kubernetes - Function code app on ARC

@andimarc does that look right?  Not sure why I don't see the equivalent ones for regular web apps defined in our assets files.

Comment (seligj95) [reply Things_You_Should_Know/kind_property.md]: added those to the list in the recent commit

Comment (andimarc) [reply Things_You_Should_Know/kind_property.md]: There's still one missing for ARC: 'app,linux,container,kubernetes'.

Aside from that, it looks right.

Comment (seligj95) [reply Things_You_Should_Know/kind_property.md]: thanks @andimarc!

**Comment by seligj95** [comment]
@btardif this should be good to go

**Review by btardif — APPROVED** [review]

**Pull request merged**